configure.ac

```diff
@@ -581,9 +581,18 @@ AS_CASE([${RKT_ACI_ARCH}],
 RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [coreos],
                   [AS_CASE([${RKT_ACI_ARCH}],
                            [aarch64],
-                                    [RKT_STAGE1_COREOS_INTERPRETER="/usr/lib/ld-linux-aarch64.so.1"],
+                                    [RKT_STAGE1_INTERPRETER="/usr/lib/ld-linux-aarch64.so.1"],
                            [amd64],
-                                    [RKT_STAGE1_COREOS_INTERPRETER="/usr/lib/ld-linux-x86-64.so.2"])])
+                                    [RKT_STAGE1_INTERPRETER="/usr/lib/ld-linux-x86-64.so.2"])])
+
+# Set the path to the interpreter binary for src flavor.
+RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [src],
+                  [AS_CASE([${RKT_ACI_ARCH}],
+                           [aarch64],
+                                    [RKT_STAGE1_INTERPRETER="/lib64/ld-linux-aarch64.so.1"],
+                           [amd64],
+                                    [RKT_STAGE1_INTERPRETER="/lib64/ld-linux-x86-64.so.2"])])
+
```

Comment:
This is fine for flavor coreos (which only has amd64+arm64), but src flavor can be built on any arch (taking the other bits for stage1 from host). I guess there should be a default case here, erroring out or reverting to the LSB loader (is that still a thing?).